app/components/view-goal/view-goal.jsx

```diff
@@ -1,8 +1,48 @@
 import React from 'react';
+import GoalTileComponent from '../goal-tile.jsx'
+import ProgressBarComponent from './progress-bar.jsx'
+
+const viewGoal = ( { currentGoal } ) => {
+  const rateGoal = (goal) => {
+    console.log("hello");
```

Comment:
Delete this and remove from jsx below as if for debugging and will take 10s to put back.